Source/WTF/wtf/PlatformHave.h

```diff
@@ -846,7 +846,7 @@
 #define HAVE_COREVIDEO_COMPRESSED_PIXEL_FORMAT_TYPES 1
 #endif
 
-#if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 120000) || PLATFORM(IOS_FAMILY)
+#if PLATFORM(MAC) || PLATFORM(IOS_FAMILY)
```

Comment:
Ditto.

Reply:
Here that would also enable it on PLATFORM(GTK), no?

Reply:
I think that we normally use `#if PLATFORM(COCOA)`, not `PLATFORM(MAC) || PLATFORM(IOS_FAMILY)` (but again, I do not remember what macCatalyst is included in).